includes/blocks/safe-svg/register.php

```diff
@@ -67,18 +67,44 @@ function render_block_callback( $attributes ) {
 	return apply_filters(
 		'safe_svg_inline_markup',
 		sprintf(
-			'<div class="safe-svg-cover" style="text-align:%s">
-				<div class="safe-svg-inside %s%s" style="width: %spx; height: %spx;">%s</div>
+			'<div class="safe-svg-cover align%s">
```

Comment:
Maybe we should keep the alignment behavior as earlier? I have no issue with new alignment updates but it may break layouts on site which are using this plugin. what do you think?

| Develop Branch | This PR |
| --- | --- |
|![Screenshot 2023-07-20 at 12 26 51 PM](https://github.com/10up/safe-svg/assets/10613171/cc08daed-db46-485d-aafb-9879acb0c952)|![Screenshot 2023-07-20 at 12 27 51 PM](https://github.com/10up/safe-svg/assets/10613171/f84f59ca-9869-47be-a664-9e75ab838150)|
|![Screenshot 2023-07-20 at 12 27 01 PM](https://github.com/10up/safe-svg/assets/10613171/0391da93-37ab-471f-b434-2222abf67c6d)|![Screenshot 2023-07-20 at 12 28 04 PM](https://github.com/10up/safe-svg/assets/10613171/77aab8f3-9540-4918-80ff-1073c8edd668)|

Reply:
@iamdharmesh Good point! Was trying to implement how the `core/image` block works since this is essentially the same, expect it being an inlined SVG. I've updated it so works as it did before.